src/python/serenity_types/pricing/derivatives/options/valuation.py

```diff
@@ -171,6 +173,18 @@ class OptionValuationRequest(CamelModel):
     individual overrides or bumps for all inputs as part of each valuation object.
     """
 
+    @validator('yield_curves', always=True)
+    def check_yield_curve_ids_or_yield_curves(cls, yield_curves, values):
+        if values.get('yield_curve_ids') and yield_curves:
+            raise ValueError("Please specify only one of 'yield_curve_ids' or 'yield_curves'")
+        return yield_curves
+
+    @validator('vol_surface', always=True)
+    def check_vol_surface_id_or_vol_surface(cls, vol_surface, values):
+        if values.get('vol_surface_id') and vol_surface:
+            raise ValueError("Please specify only one of 'vol_surface_id' or 'vol_surface'")
+        return vol_surface
```

Comment:
I think we still want validation that user can only specify one and only one of
Note, this validator allow both to be None

- yield_curve_ids vs yield_curves 
- vol_surface_id vs vol_surface